tests/unit/command/test_cache.py

```diff
@@ -0,0 +1,40 @@
+import os
+import pytest
+from mock import MagicMock
+
+from dvc.config import Config
+
+
+@pytest.mark.skipif(os.name == "nt", reason="Linux only")
+def test_path_replacement():
+
+    config_filename = "./config/file"
+
+    # Extract the "rel()" function from Config._save_paths
```

Comment:
This is hacky, you could've just moved `rel` helper outside and tested it separately :slightly_smiling_face:

Reply:
It is quite. I thought about moving it out, but it needs to have `conf_dir` embedded into it. I haven't dug into `Schema` but I assume I'm not allowed to change the signature of `func`. 

Then again, I could use a partial which would be rather less hacky. I'll do that

Reply:
@charlesbaynham Oh, missed that one. Another way to test this would be to use func tests for `config`. Instead of unit testing this particular helper. I probably should've suggested it initially.

Reply:
Ok, one more thing that I didn't notice initially: this change will also affect other paths in the config. Trying to think if that could be bad in some application...